test/SparkLendMainnetIntegration.t.sol

```diff
@@ -343,4 +322,27 @@ contract SparkLendMainnetIntegrationTest is Test {
         assertEq(aaveOracle.getSourceOfAsset(WSTETH), address(oracle));
     }
 
+    // ---- Helper Functions ----
```

Comment:
Should maintain consistency with rest of codebase with

```
    /**********************************************************************************************/
    /*** Declarations and Constructor                                                           ***/
    /**********************************************************************************************/
    ```

styling

Reply:
done